react/features/toolbox/components/buttons/VideoQualityButton.web.js

```diff
@@ -0,0 +1,149 @@
+import AKInlineDialog from '@atlaskit/inline-dialog';
```

Comment:
I see. In general I would argue that ideally every feature should be a black box and could be turned on and off without touching any other feature. In the case of the toolbar it would mean that the toolbox feature should give us a way to register any button into it. We're not in an ideal world though and have some legacy code to deal with, so I'll think about it a little bit. I agree that a circular dependency isn't an option.